### ISSUE TITLE
`dart test` shows mentions `pub run test`

### ISSUE THREAD

**Issue by mit-mit** [dart-lang/test#1298]
When the test directory is missing:

```
$ dart test
Precompiling executable... (13.4s)
Precompiled test:test.
No test files were passed and the default "test/" directory doesn't exist.

Usage: pub run test [files or directories...]
```

**Comment by grouma** [comment]
We'll need to update the minimum SDK constraint in `package:test` if we start using this verbiage. When did `dart test` land?

**Comment by jakemac53** [comment]
Note that changing the message will be equally confusing for users who use `pub run test` today.

If the goal is to push everybody to `dart test` then we should add a message if we detect you are running via `pub run test` (assuming that is possible), which directs you to `dart test`.

**Comment by natebosch** [comment]
Fixed in #1554
